### ISSUE TITLE
A question about sts(Semantic Text Similarity) data load

### ISSUE THREAD

**Issue by jwc19890114** [pytorch/text#687]
## ❓ Questions and Help

**Description**

i have a question when i tring to use torchtext  in train DSSM (an sts model).
I don't know how to construct the data.
we have  2 lists (query list and doc list) and a label list to send in the model, how to use torchtext?

**Comment by zhangguanheng66** [comment]
If you want to use `DataLoader` in `torch.utils.data`, you could write the dataset pipeline with the new abstraction. Copy/post most code in text classification datasets ([link](https://github.com/pytorch/text/blob/master/torchtext/datasets/text_classification.py)) should be enough. Depending on the format, you need to load the text data into memory. The text classification datasets come with label/text in a single line. Other than that, the data should be very similar.
